README.md

```diff
@@ -0,0 +1,6 @@
+# vaultkeeper
+
+## Installing the Package
+
+Install the package for development with the following command:
+`pip install -e . --process-dependency-links`
```

Comment:
For development you would want the test dependencies. So:
```
pip install -e .[testing] --process-dependency-links
```

Also, while markdown is (IMO) nicer to write than restructuredtext, rst is the format supported by PyPI. So if you want your text to be formatted in the package listing then this has to be rst, unfortunately.